javascript-timers/timeout.js

```diff
@@ -0,0 +1,7 @@
+const heading = document.querySelector('h1');
+
+function myMessage() {
+  heading.innerHTML = 'hello there';
```

Comment:
Should be using `textContent` instead. `innerHTML` has privacy & security problems